test/labeling/model/test_label_model.py

```diff
@@ -548,6 +549,23 @@ def test_symmetry_breaking_multiclass(self):
         self.assertEqual(label_model.mu.data[0, 0], 0.15)
         self.assertEqual(label_model.mu.data[1, 1], 0.3)
 
+    def test_lambda_lr_save(self):
+        label_model = LabelModel(cardinality=2)
+        L_train = np.array(
+            [[1, 0, 1], [1, 1, 1], [1, 0, 1], [1, 1, 0], [0, 0, 0], [1, 1, 1]]
+        )
+        label_model.fit(
+            L_train=L_train,
+            n_epochs=500,
+            lr=0.001,
+            lr_scheduler="linear",
+            optimizer="adam",
+            log_freq=100,
+            seed=123,
+        )
+        label_model.save("./test_save.pickle")
```

Comment:
Not a bad idea to include a comment why this test exists (i.e., it's sort of strange to save a model then delete it without reloading or doing anything else with it). Can mention that this model was not previously pickleable due to a lambda function in the schedule (leave breadcrumbs for future readers of the code)